client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.module.css

```
@@ -57,6 +57,14 @@
     /* … */
 }
 
+.results {
+  padding-left: 5px;
+  padding-top: 5px;
```

Instead of `padding-top`, can we do `align-self: center`?
```suggestion
  align-self: center;
```